module-5/src/utils/parseRequestBody.js

```diff
@@ -0,0 +1,16 @@
+export const parseRequestBody = (req) =>
+  new Promise((resolve, reject) => {
+    let body = "";
+
+    req.on("data", (chunk) => {
+      body += chunk.toString();
+    });
+
+    req.on("end", () => {
+      resolve(JSON.parse(body));
```

Comment:
When parsing the request body, JSON parsing errors can occur if the client sends invalid JSON. Wrap this line in a `try-catch` block to catch and handle these errors, possibly returning a `400 Bad Request` status to the client

But task description states that `Input validation` can be skipped so ignore this comment